### ISSUE TITLE
Translate

### ISSUE THREAD

**Issue by Sledav** [wolfgangimig/itol#46]
I've translated some label to Russian language in file IssueServiceImpl.js. How I can translate all labels in  addin?

**Comment by wolfgangimig** [comment]
You would have to compile the Add-in. See https://github.com/wolfgangimig/itol/wiki/Developer-Information for a short documentation.
In class Globals, function getResourceBundle, you would have to replaces the res_en.properties file with your Russian language properties file.

**Comment by Sledav** [comment]
Thanks! I translated all labels in AddIn, but not labels are available in the res_en.properties file.

**Comment by wolfgangimig** [comment]
Could you explain this in some more words, please give an example.

**Comment by Sledav** [comment]
For example, \itol-addin\src\com\wilutions\itol\AttachmentTableViewHandler.java
rows 68:
TableColumn<Attachment, String> fileNameColumn = new TableColumn<>("Name");
in Russian 
TableColumn<Attachment, String> fileNameColumn = new TableColumn<>("Имя файла");

and 102
TableColumn<Attachment, Long> contentLengthColumn = new TableColumn<>("Size");
Russian
TableColumn<Attachment, Long> contentLengthColumn = new TableColumn<>("Размер");

**Comment by wolfgangimig** [comment]
Obviously, I was a little careless at some UI definitions. You will have to replace the literals with e.g.:

```
TableColumn fileNameColumn = new TableColumn<>(resb.getString("tabAttachments.nameColumn"));
```

Then add a line to the resource file:

```
tabAttachments.nameColumn=Имя файла
```

Regards

**Comment by Sledav** [comment]
Thanks.  Now I translated all labels in AddIn.
I found a small errors in
https://github.com/wolfgangimig/itol/blob/master/itol-addin/src/com/wilutions/itol/DlgConfigure.java#L98

Please replace DlgConnect.LogLevel.Info with DlgConfigure.LogLevel.Info
